lib/wings/navigators/child_collections_navigator.rb

```diff
@@ -0,0 +1,35 @@
+# Navigate from a resource to the child collections in the resource.
+module Wings
+  class ChildCollectionsNavigator
+    # Define the queries that can be fulfilled by this navigator.
```

Comment:
These set of classes and methods define navigator services.  An alternate pattern would be to define valkyrie custom queries.  

Considerations:
* valkyrie custom queries is an established pattern
* navigators use a naming scheme and setup similar to custom queries
* navigators do not make any direct queries of the backend.  They only use methods in the primary QueryService.

Reply:
I really like this pattern.